pkg/validation/validate/slice_validator.go

```diff
@@ -54,7 +54,7 @@ func (s *schemaSliceValidator) Validate(data interface{}) *Result {
 	size := val.Len()
 
 	if s.Items != nil && s.Items.Schema != nil {
-		validator := NewSchemaValidator(s.Items.Schema, s.Root, s.Path, s.KnownFormats, s.Options.Options()...)
+		validator := s.Options.subIndexValidator(0, s.Items.Schema)
```

Comment:
this is wrong, will give you the wrong index in results

Reply:
actually not sure, it is wrong. But least it is surprising to pass zero. Why the index param in the first place?